Mathlib/MeasureTheory/Integral/Lebesgue.lean

```diff
@@ -769,12 +769,21 @@ theorem lintegral_indicator (f : α → ℝ≥0∞) {s : Set α} (hs : Measurabl
   refine ⟨⟨φ.restrict s, fun x => ?_⟩, le_rfl⟩
   simp [hφ x, hs, indicator_le_indicator]
 
+lemma setLintegral_indicator (f : α → ℝ≥0∞) {s t : Set α} (hs : MeasurableSet s) :
```

Comment:
It looks like `setLIntegral` is a more popular capitalization. Could you please change it in all 3 lemmas? Otherwise LGTM. Thanks!
bors d+

Reply:
Hello Yury,

Apologies for the delay - just made changes as you suggested. Thanks!